broker-plugin/broker-cli/src/main/java/io/enmasse/broker/cli/Cli.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2016-2018, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.enmasse.broker.cli;
+
+
+import io.enmasse.amqp.Artemis;
+import io.enmasse.amqp.SyncRequestClient;
+import io.vertx.core.Vertx;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class Cli {
+
+    private static final Logger log = LoggerFactory.getLogger(Cli.class.getName());
+
+    private SyncRequestClient brokerClient;
+    private Artemis artemis;
+    private String certDir = "/opt/apache-artemis/custom/enmasse-certs";
+
+    private void start() {
+
+        Vertx vertx = Vertx.vertx();
+        ClientFactory clientFactory = new ClientFactory(vertx, certDir);
+
+        try {
+            brokerClient = clientFactory.connectBrokerManagementClient("localhost", 5673);
+            artemis = new Artemis(brokerClient);
+
+        } catch (NullPointerException e) {
+            log.error("Failed to create/connect to client: " + e.toString());
+        } catch (Exception e) {
+            log.error("Error: " + e.toString());
+        }
+    }
+
+    private void closeClient() {
+        try {
+            brokerClient.close();
+        } catch (Exception e) {
+            log.error("Error closing client. " + e.toString());
+        }
+    }
+
+    private String executeCommandWithReturn(String command, String... queue) throws InvocationTargetException, IllegalAccessException, NoSuchMethodException {
+        log.info("Invoking Method with return");
+        String result ="";
+            Method method = artemis.getClass().getMethod(command, String.class);
+            log.info("Return type: " + method.getReturnType());
+            if (method.getReturnType().toString().trim().equals("void")) {
+                method.invoke(artemis, queue);
+                result = "success";
+            } else {
+                result = method.invoke(artemis, queue).toString();
+            }
+        return result;
+    }
+
+    public static void main(String[] args) {
+
+        Cli cli = new Cli();
+        cli.start();
+
+        String command = args[0];
+        String queue = args[1];
+
+
+        log.info("Executing " + command );
+        String result;
+        try {
+            result = String.valueOf(cli.executeCommandWithReturn(command, queue));
+
+            log.info("Result: " + result);
+        } catch (InvocationTargetException e) {
+            log.info("Error executing command: " + e.toString());
+        } catch (IllegalAccessException e) {
+            log.info("Error executing command: " + e.toString());
+        } catch (NoSuchMethodException e) {
+            log.info("Error executing command: " + e.toString());
+        }
+        cli.closeClient();
+        System.exit(0);
```

Comment:
Programs that are designed to be used on the command line should return a non-zero exit status in failure cases.